kvmagent/kvmagent/test/test_snapshots.py

```
@@ -1,9 +1,11 @@
 from kvmagent.test.utils import vm_utils, network_utils, pytest_utils
 from kvmagent.test.utils.stub import *
 from zstacklib.test.utils import env, misc
-from zstacklib.utils import uuidhelper
+from zstacklib.utils import uuidhelper, xmlobject
```

Ruff (0.12.2)

`zstacklib.utils.xmlobject` imported but unused

Remove unused import: `zstacklib.utils.xmlobject`

(F401)